.github/workflows/test.yml

```diff
@@ -33,34 +33,17 @@ jobs:
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 3.2.2
           # runs 'bundle install' and caches installed gems automatically
           bundler-cache: true
 
       - name: Set up Node
         uses: actions/setup-node@v4
         with:
-          node-version: 14
-
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-
-      - name: Cache node modules
-        uses: actions/cache@v4
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-
-      - name: Install linux dependencies
-        run: |
-          sudo apt-get -yqq install libpq-dev build-essential libssl-dev
+          node-version: 22.3.0
+          cache: 'yarn'
```

Comment:
`cache: 'yarn'` will load and save the cache, but doesn't actually install the dependencies, for some reason